pkg/inject/sidecar.go

```diff
@@ -367,6 +391,20 @@ func HasJaegerAgent(dep *appsv1.Deployment) (bool, int) {
 	return false, -1
 }
 
+// IsJaegerPortAvailable checks whether deployment is already using some port
+func IsJaegerPortAvailable(port int32, dep *appsv1.Deployment) bool {
```

Comment:
This is currently being used only by the package itself, I would make it private for now.